src/components/TalkToUs/index.js

```diff
@@ -36,7 +52,11 @@ const TalkToUs = () => {
             <div css={closeSymbol} />
           </button>
         </div>
-        <div css={popoverBody}>{state && <LeaveMessage />}</div>
+        <div css={popoverBody}>
+          {state && (
+            <LeaveMessage onFormStatusChange={handleFormStatusChange} />
```

Comment:
This prop exposes too much detail. I think a more idiomatic way to achieve what you want is to use prop like `onSubmit`. Please change it